features/general/file-manager.md

```diff
@@ -0,0 +1,105 @@
+# File Manager of ERP.net
+
+## Overview
+
+Oftentimes users have to search through complex directories to find a file they are looking for. 
+Especially when the file is **not associated** to a **specific document or record** in the ERP system, buth is a rather more **generic document** for business use, such as documentation regarding training courses, internal or external regulations, company practices, handbooks, and so on.  
+
+Users tend to keep such files in an external storage, like local drives or cloud platforms. But this often leaves a gap between navigating through internal ERP entities and external file storage.  
+
+A **File Manager inside the ERP system** is a **feasible and productive way** to **close this gap** and **utilize the ERP system's full potential**.  
+
+Your Business Management solution **@@name** comes with the updated functionality **File Management v2.0**.  
```

Comment:
Your Business Management solution - too heavy. Maybe, just skip it.

Reply:
A simple way of stating the thought was used.